streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountJob.java

```diff
@@ -80,9 +68,9 @@ public static void main(String[] args) throws Exception {
                         return new KeyValue<String, String>(value, value);
                     }
                 })
-                .countByKey(stringSerializer, stringDeserializer, "Counts");
+                .countByKey(SerDes.STRING(), "Counts");
 
-        counts.to("streams-wordcount-output", stringSerializer, longSerializer);
+        counts.to("streams-wordcount-output", SerDes.STRING().serializer(), SerDes.LONG().serializer());
```

Comment:
Another example where the user should only need to pass in the "top-level" serde -- we can pull out the serializer part behind the scenes, no need to bother the user with doing that.

```java
counts.to("streams-wordcount-output",
    SerDes.STRING(), /* key serde */
    SerDes.LONG() /* long serde */
);
```